pipeline/io.py

```diff
@@ -0,0 +1,99 @@
+import base64
+
+
+class File:
+    """
+    IO file descriptor. Allows to manipulate files between client and server.
+
+    Example: (Transfer an audio file)
+
+        '''
+        On client
+        '''
+
+        from pipeline.io import File
+
+        FILE_PATH = "files/audio.flac"
```

Comment:
Having to encode the extension leads to a bit of a dance. Could we skip trying to ship the extension entirely? The user's function anyhow has to do the `File.file_get_bytes` call, so if they really need the extension they can include it again there.